runtime/service/npud/core/DevManager.cc

```diff
@@ -97,5 +98,27 @@ std::shared_ptr<Backend> DevManager::getBackend()
   return _dev->loader->getInstance();
 }
 
+int DevManager::createContext(int deviceId, int priority, NpuContext **npuContext)
+{
+  auto backend = getBackend();
+  if (!backend)
+  {
+    return NPU_STATUS_ERROR_OPERATION_FAILED;
+  }
```

Comment:
Is there any good way to handle it? This phrase is repeated over and over again. It would be nice to use `inline` or `define`.... let me check it again..

Reply:
For DC score, I recommend using a ternary operator.

Reply:
I've already considered using a `ternary operator`.. But this can't be used here. Anyway, I will find a better way for this. :)